### PR TITLE
Drop NMT mode

### LINKED ISSUE #28: Please remove *NMT modes from Textra translation mode option list.
**NICT has deleted all NMT translation modes on Nov. 10, so OmegaT can no longer use those modes.**
Please see https://mt-auto-minhon-mlt.ucri.jgn-x.jp/content/menu/

**Remove NMT modes from Textra Configuration window.**

**There is no alternative solutions, but this request doesn't have very high priority.**

### PR DESCRIPTION
**Pull request by miurahr**
Fix #28

Signed-off-by: Hiroshi Miura <miurahr@linux.com>

### CODE REVIEW

**Comment by miurahr** [comment]
This is a snapshot build with the patch.

[omegat-textra-plugin-0.10.1-SNAPSHOT.zip](https://github.com/miurahr/omegat-textra-plugin/files/5527683/omegat-textra-plugin-0.10.1-SNAPSHOT.zip)

File: src/main/java/tokyo/northside/omegat/textra/TextraOptions.java [hunk 1]
```diff
@@ -23,24 +23,13 @@
...
-                Arrays.asList(Mode.generalN, Mode.patentN, Mode.patent_claimN),
-                Arrays.asList("ko", "ja")));
-        combination.addAll(createCombinations(
-                Arrays.asList(Mode.generalN, Mode.patentN, Mode.patent_claimN),
-                Arrays.asList("de", "ja")));
-        combination.addAll(createCombinations(Mode.generalN, "en",
-                Arrays.asList("de", "fr", "pt", "fr", "id", "my", "th", "vi", "es")));
         combination.addAll(createCombinations(
                 Arrays.asList(Mode.generalNT, Mode.patentNT),
                 Arrays.asList("ja", "en", "zh-CN", "zh-TW")));
```
Comment (miurahr) [inline src/main/java/tokyo/northside/omegat/textra/TextraOptions.java:28]: We need to add more new NT mode language combinations here.

**Comment by MieSato** [comment]
Thank you, it worked! 
In addition, I found that I don't need to reconfigure Textla setting on restarting OmegaT (unlike before)!! 
By the way, can you add NT+ mode as well?
NICT say that we can use NT+ for OmegaT.

**Comment by miurahr** [comment]
>Can you add NT+ mode as well?
>NICT say that we can use NT+ for OmegaT.

I'd like to work  it on another issue ticket, (afterword).

**Pull request merged**